### DISCUSSION TITLE
Auth0 log events to Seq

### DISCUSSION THREAD

**Discussion by mskog2023** [datalust/seq-tickets#2065] [Help] [answered]
I'm trying to integrate Auth0 with Seq. For this I have built a simple API that receives log events from Auth0 and forwards them to Seq using the HTTP API. Is there a way to keep the timestamp that I'm receiving from Auth0 in Seq? In my payload I try to set the timestamp but in Seq it shows up as a ClientTimestamp property. Is this expected behaviour or am I doing something wrong?

**Comment by nblumhardt** [comment]
Hi @mskog2023, thanks for your message.

Seq does this when events arrive more than an hour or so into its "future".

My guess is that you might not be including the time zone correctly in the timestamps you're sending to Seq as `@t`. These need to have the time zone specified as either `Z` (UTC) or `+HH:MM`/`-HH:MM`.

Another possibility is that the clock on the Seq server machine is wrong.

If you can share an example of your generated payload, and the events in Seq, we'll be able to assist.

Hope this helps,
Nick

**Reply by mskog2023** [reply, marked as answer]
Thank you, @nblumhardt! I managed to solve the issue. It was the time zone information that was causing it.
